components/proxy/src/main/java/com/hotels/styx/admin/AdminServerBuilder.java

```diff
@@ -200,6 +201,8 @@ private HttpHandler adminEndpoints(StyxConfig styxConfig, StartupConfig startupC
         httpRouter.aggregate("/admin/servers", serverHandler);
         httpRouter.aggregate("/admin/servers/", serverHandler);
 
+        httpRouter.aggregate("/metrics", new PrometheusHandler(environment.meterRegistry()));
+
```

Comment:
All other admin interface endpoints have an `/admin` prefix, but the new `/metrics` does not.  The new metrics endpoint should be consistent with the rest and therefore should be `/admin/metrics`.

Reply:
`/metrics` was chosen because it's the default path for prometheus metrics.